solidity/test/rewards/TestECDSARewards.js

```diff
@@ -94,9 +105,17 @@ describe("ECDSARewards", () => {
     })
 
     it("should equal expected allocation in third interval", async () => {
-      const secondIntervalEnd = await timeJumpToEndOfInterval(1)
-      await keepFactory.stubBatchOpenFakeKeeps(40, secondIntervalEnd)
-      await timeJumpToEndOfInterval(2)
+      const endOf = await rewardsContract.endOf(1)
+      let now = await time.latest()
+      if (now.lt(endOf)) {
+        await time.increaseTo(endOf.addn(1))
+        now = await time.latest()
+        await keepFactory.stubBatchOpenFakeKeeps(40, now)
+      } else {
+        await keepFactory.stubBatchOpenFakeKeeps(40, endOf.addn(1))
+      }
```

Comment:
I think it can be simplified to:
```
const secondIntervalEnd = await rewardsContract.endOf(1)
await keepFactory.stubBatchOpenFakeKeeps(40, secondIntervalEnd.addn(1))
```

Reply:
Right, done